### ISSUE TITLE
Add the option of exporting to keep in ToDo list

### ISSUE THREAD

**Issue by Raghwendra-Dey** [Raghwendra-Dey/productivity_meter#55]
Add a Button `Export to Keep` in the ToDo list, which first ask the user about to login to his google account, and then create a new TODO list in his google keep.
Later, we can also add an optionn to `import from Keep` which imports any list in keep with checkboxes into this.

**Comment by Raghwendra-Dey** [comment]
@Bbiswabasu would you like to do this?

**Comment by Bbiswabasu** [comment]
Yes I will take it. But I do not have much experience in this matter. So, I may need some time to complete it

**Comment by Raghwendra-Dey** [comment]
Great!! You take your time, no issues. You need to look into how to redirect to google login, then work something with keep api.
Would you be doing both `export` and `import` at the same time, or just `export` for now, and `import` later?
I suggest doing one at a time would be better.

**Comment by Raghwendra-Dey** [comment]
Oops! seems like, theres no keep api as such. You might need to look for any alternatives.

**Comment by Bbiswabasu** [comment]
Okay let me start the task. I would like to do both import and export. Still, I will inform you if I face problem in doing any of these tasks

**Comment by Bbiswabasu** [comment]
> Oops! seems like, theres no keep api as such. You might need to look for any alternatives.

Hmm there isn't any. @Raghwendra-Dey How about uploading the ToDo list to Google Drive?

**Comment by Raghwendra-Dey** [comment]
> > Oops! seems like, theres no keep api as such. You might need to look for any alternatives.
> 
> Hmm there isn't any. @Raghwendra-Dey How about uploading the ToDo list to Google Drive?

Could you try to use [this](https://github.com/kiwiz/gkeepapi) ? The problem is the python in js thing. Is there anything you could do about that?

**Comment by Bbiswabasu** [comment]
I am not sure how we can use this from js

**Comment by Raghwendra-Dey** [comment]
> I am not sure how we can use this from js

Okay, for the time being, you work to export this to a google doc. then later on we may modify it for keep, if we find a way to work with that api.

**Comment by Bbiswabasu** [comment]
I checked API for Docs and it seems like we need some backend stuff. I don't think we can make calls just from javascript

**Comment by Raghwendra-Dey** [comment]
@Bbiswabasu you may make use of any js framework, for it. I think there are work arounds as well in js itself, though not sure about them. Also, after the localStorage thing is done, we would be introducing jQuery to the project, its work is already started after #62 , so, look if jQuery helps.

**Comment by Raghwendra-Dey** [comment]
Hey @Bbiswabasu have you looked into the possible ways to do this. Please explain any problems you might be facing.

**Comment by Bbiswabasu** [comment]
@Raghwendra-Dey I read the documentation of the API. Python was mentioned as a prerequisite to provide server. I searched but didn't find a way to do the task with js

**Comment by Raghwendra-Dey** [comment]
@Bbiswabasu okay! then what about importing to google docs? Any issues with that?
